### PR TITLE
503 handling

### PR DESCRIPTION
**Pull request by paveg**
## Overview

https://lgtm.in/g returned 503.

screen shot 2018-07-31 at 16 27 07

I corresponded.

when https://www.lgtm.in/ returns 503, displays the html of the following.

```html
<h1 align='center'>LGTM</h1>
```

## Example

<h1 align='center'>LGTM</h1>

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -19,7 +19,7 @@ GEM
     colored2 (3.1.2)
     cork (0.3.0)
       colored2 (~> 3.1)
-    danger (5.6.2)
+    danger (5.6.4)
```
Comment (paveg) [inline Gemfile.lock:22]: and bump up danger as well.

**Comment by paveg** [comment]
@leonhartX Please, review it.

**Comment by leonhartX** [comment]
LGTM, thanks!

**Comment by leonhartX** [comment]
1.0.2 released

**Pull request merged**